index.js

```diff
@@ -0,0 +1,127 @@
+
+fetch('http://localhost:3000/players')
+.then(resp => resp.json())
+.then(data => {
+  // const players = data
+  console.log(data)
+
+  data.forEach(renderPlayers)
+})
+.catch(error => {
+  alert("Bad things! Ragnarők!")
+  console.log(error.message)
+})
+
+
+const playerContainer = document.querySelector('.player-container')
+
+
+function renderPlayers(player) {
+  const playerDiv = document.createElement('div')
+
+  playerDiv.className = 'player'
+  playerDiv.dataset.number = player.id
+
+  playerDiv.innerHTML = `
+    <h3>${player.name} (<em>${player.nickname}</em>)</h3>
+
+    <img src=${player.photo} alt=${player.name}>
+
+    <p>${player.likes} Likes</p>
+
+    <button class="like-button">❤️</button>
+  `
+
+  playerContainer.append(playerDiv)
+}
+
+const playerForm = document.querySelector('#new-player-form')
+
+playerForm.addEventListener("submit", event => {
+  event.preventDefault()
+
+  const form = event.target
+
+  const playerObj = {
+    number: form.number.value,
+    name: form.name.value,
+    nickname: form.nickname.value,
+    photo: form.photo.value,
+    likes: 1000
+  }
+
+  updateData(playerObj, form)
+})
+
+function updateData(playerObj, form) {
+  const configObj = {
+    method: "POST",
+    headers: {
+    "Content-Type": "application/json",
+    "Accept": "application/json"
+    },
+    body: JSON.stringify(playerObj)
+  }
+
+  console.log(configObj)
+
+  fetch('http://localhost:3000/players', configObj)
+  .then (resp => resp.json())
+  .then (data => {
+    renderPlayers(playerObj)
+    form.reset()
+    console.log(data)
+  })
```

Comment:
Here, you'd want to use `renderPlayers(data)` instead of using `playerObj` - `data` will be the object returned from the server, which will have an id for the player. Since you need the id in the `renderPlayers` function, it won't quite work when you just use the `playerObj` from the form.